packages/components/src/InputGroup/InputGroup.test.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { cleanup } from "@testing-library/react";
+import { InputGroup } from ".";
+
+afterEach(cleanup);
+
+it("renders a vertical InputGroup", () => {
+  const tree = renderer.create(<InputGroup>Test content</InputGroup>).toJSON();
```

Comment:
We can probably get away with snapshot tests here for now. Perhaps a simple one in each direction then a more complicated combination?

Reply:
Done, also included an invalid nesting case.